neonvm/controllers/virtualmachine_controller.go

```diff
@@ -364,8 +374,30 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 		vmRunner := &corev1.Pod{}
 		err := r.Get(ctx, types.NamespacedName{Name: virtualmachine.Status.PodName, Namespace: virtualmachine.Namespace}, vmRunner)
 		if err != nil && apierrors.IsNotFound(err) {
+			// Check if the ssh secret already exists, if not create a new one
```

Comment:
This block gets too nested, maybe extract it to a function?

Reply:
I wanted to write it the same way the rest of code is written in the controller. I agree we should refactor controller's codebase with the right set of abstractions but I'd leave that for another task. I'll make an issue for it.

Reply:
Opened https://github.com/neondatabase/autoscaling/issues/733